src/Stellar/Signer.h

```diff
@@ -29,6 +31,7 @@ class Signer {
 
   private:
     void encodeAddress(const Address& address, Data& data) const;
+    void encodeAsset(const std::string& issuer, const std::string& alphamum4, const std::string& alphamum12, Data& data) const;
```

Comment:
what is `alphanum4` or `alphanum12`, asset id ?

Reply:
Yes, assets have a 4-char or 12-char ID.  https://www.stellar.org/developers/guides/concepts/assets.html

Reply:
it's safe to check asset id by string length? so that we can use one asset id variable